CMakeLists.txt

```diff
@@ -52,8 +52,10 @@ endif()
 # Specify the C++ standard
 set(CMAKE_CXX_STANDARD 14)
 
+add_subdirectory(third_party)
 add_subdirectory(src)
 
+add_subdirectory(unit_tests)
```

Comment:
Move this lower down so you don't need the `enable_testing()` statement in unit_tests/CMakeLists.txt

Reply:
Done